api/docs/release.dox

```diff
@@ -229,6 +229,8 @@ Further non-compatibility-affecting changes include:
    of built basic blocks.
  - Added drreg_reservation_info_ex(), drreg_statelessly_restore_app_value(),
    and drreg_is_instr_spill_or_restore().
+ - Module tracking allocates an entry per segment for each loaded module.
```

Comment:
Please include `drmodtrack` so we know which library/component this is: the core API provides module information as well.

Reply:
Done.